pkg/injector/envoy_config_test.go

```diff
@@ -162,7 +162,6 @@ var _ = Describe("Test Envoy sidecar", func() {
 		It("creates correct Envoy sidecar spec", func() {
 			mockConfigurator.EXPECT().GetEnvoyLogLevel().Return("debug").Times(1)
 			actual := getEnvoySidecarContainerSpec(containerName, envoyImage, nodeID, clusterID, mockConfigurator)
-			Expect(len(actual)).To(Equal(1))
```

Comment:
`getEnvoySidecarContainerSpec()` returns a single `Cluster` now - no need to check the size, it is no longer a list